tests/tests/data-prealloc.sh

```diff
@@ -197,7 +180,12 @@ for sides in 0 1 2 3; do
 done
 
 echo before:
-print_logical_extents "$prefix"
+scoutfs get-fiemap "$prefix" | tr -s ',' '\t' | \
+	awk '($1 != "entries:") {
```

Comment:
Removing the function that housed this fiddly awk and then  copying and pasting it where the function used to be called is a bit much.  Move the duplicate translation of the output up into that function.